File: MORM.CSharp/Classes/Conexao.cs
namespace MORM.Reppositorio.Classes
{
    public class Conexao
    {
        public Conexao(Parametro parametro)
        {
            Parametro = parametro;
        }
        
        public Parametro Parametro { get; private set; }
        
        public void ExecComando(string cmd)
        {
            var command = Connection.CreateCommand();
            command.CommandText = cmd;
            command.ExecuteNonQuery();
        }

        public DataReader GetConsulta(string sql)
        {
            var command = Connection.CreateCommand();
            command.CommandText = sql;
            return command.ExecuteReader();
        }        
    }
}

File: MORM.Java/Classes/Conexao.java
package MORM.Java.Classes;

public class Conexao
{
    public Conexao(Parametro parametro)
    {
        Parametro = parametro;
    }
    
    public Parametro Parametro;
    private Connection Connection;
    
    public DataReader GetConsulta(String sql)
    {
        Statement stm = Connection.createStatement();
        ResultSet rs = stm.executeQuery(sql);
        return rs;
    }
    
    public void ExecComando(String cmd)
    {
        Statement stm = Connection.createStatement();
		stm.execute(cmd);
    }
}